ez/ez.go

```diff
@@ -38,8 +41,15 @@ func WithFlagConfig(flagConfig *flag.NameConfig) Option {
 
 // WithWatchingConfigFile allows to watch the config file by using the watching
 // file source
-func WithWatchingConfigFile() Option {
-	return func(d *dialsOptions) { d.watch = true }
+func WithWatchingConfigFile(enabled bool) Option {
```

Comment:
Let's extend the docstring a touch to mention that the default is disabled. (without this option)